draft-ietf-rats-corim.md

```diff
@@ -2508,6 +2524,26 @@ If no entry is found, the comparison MUST return false.
 Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after equivalence has been found according to {{sec-comid-integrity-registers}}.
 Note that it is not required for all the entries in the candidate entry to be used during matching: the condition ECT could consist of a subset of the device's register space. In TPM parlance, a TPM "quote" may report all PCRs in Evidence, while a condition ECT could describe a subset of PCRs.
 
+##### Comparison for linear-privlevel entries
+
+The ACS entry value stored under `measurement-values-map` codepoint 15 is a linear privilege level, which must have type `linear-privlevel-type-choice`.
+
+If the entry `linear-privlevel-type-choice` is an `int`, name it PRIV and compare as follows.
+
+*  If the condition ECT value for `measurement-values-map` codepoint 15 is an `int` then an equality comparison is performed on the `int` components.
+The comparison MUST return true if the value of PRIV is equal to the `int` value in the condition ECT.
+
+*  If the condition ECT value for `measurement-values-map` codepoint 15 is an `int-range` tagged with #6.564 then a range inclusion comparison is performed.
+The comparison MUST return true if the value of PRIV is greater than or equal to the `min` value in the condition ECT AND the value of PRIV is less than or equal to than the `max` value in the condition ECT
+
+If the entry `linear-privilege-type-choice` is an `int-range` or `int-range` tagged with #6.564, then comparison with the pair of `opt-int` values MINPRIV and MAXPRIV is as follows.
```

Comment:
I find it hard to understand the logic for 2539 beneath.. Is there a way to simplify/clarify it better??